slice.go

```diff
@@ -693,3 +694,64 @@ func Splice[T any, Slice ~[]T](collection Slice, i int, elements ...T) Slice {
 
 	return append(append(append(output, collection[:i]...), elements...), collection[i:]...)
 }
+
+// Pull removes all given values from slice using SameValueZero for equality comparisons.
```

Comment:
```suggestion
// Pull removes all given values from slice using SameValueZero for equality comparisons.
```

I'm sorry but you are not using SameValueZero to check the item to filter. It looks like a copy pasta from Splice

Reply:
Done